programs/random/gen_random_ctr_drbg.c

```diff
@@ -53,21 +53,39 @@ int main( int argc, char *argv[] )
 {
     FILE *f;
     int i, k, ret;
+#if defined(MBEDTLS_PLATFORM_C)
+    mbedtls_platform_context platform_ctx;
+#endif
     mbedtls_ctr_drbg_context ctr_drbg;
     mbedtls_entropy_context entropy;
     unsigned char buf[1024];
 
+#if defined(MBEDTLS_PLATFORM_C)
+    if( ( ret = mbedtls_platform_setup( &platform_ctx ) ) != 0 )
+    {
+        mbedtls_printf( " failed\n  !  mbedtls_platform_setup returned %d\n\n", -ret );
```

Comment:
The return value is displayed as a negative here, but as positive at line 92. We should be self-consistent within the same program.